### PR TITLE
Fix the create release pr script to include proper regex

### PR DESCRIPTION
**Pull request by mpryc**
Regex for the Install.md should be replaced with regex(es) to the Development.md, because that part of docs was moved.

Signed-off-by: Michal Pryc <mpryc@redhat.com>

@redhat-cop/mdt

### CODE REVIEW

File: scripts/create_release_pr [hunk 1]
```diff
@@ -111,8 +111,10 @@ else
   sed -i "s/^version:.*/version: $NEXT_CHART_VER/g" "$PELORUS_CHART"
 fi
 
-# Update branch in the Install documentation
-sed -i "s/--branch v.* /--branch v$V_MAJOR.$V_MINOR.$V_PATCH /g" "$INSTALL_DOC"
+# Update branch in the Development documentation
+sed -i "s/\(release number, for example \`\).*\(\`\.\)/\1v$V_MAJOR.$V_MINOR.$V_PATCH\2/g" "$INSTALL_DOC"
```
Comment (mateusoliveira43) [inline scripts/create_release_pr:115]: can we add a flag or something to check if the operation was successful?

Comment (mpryc) [reply scripts/create_release_pr:115]: No, sed does not have control over its exit code. We could in theory check by using additional grep or there is hack to "echo" sort of fake exit code directly from sed, however I don't think it's any use here.

We could always check by running `git diff` or `git status` and see if files were changed, but that's really over-design.

Comment (mateusoliveira43) [reply scripts/create_release_pr:115]: ok

**Review by mateusoliveira43 — APPROVED** [review]

**Review by mateusoliveira43 — CHANGES_REQUESTED** [review]
I have two question: when running `make release`

- charts/pelorus/subcharts/exporters/Chart.yaml should be updated?
- charts/pelorus/subcharts/exporters/templates/_buildconfig.yaml `spec.source.git.ref` default updates to chart or pelorus version?

**Comment by mpryc** [comment]
> I have two question: when running `make release`
> 
> * charts/pelorus/subcharts/exporters/Chart.yaml should be updated?
> * charts/pelorus/subcharts/exporters/templates/_buildconfig.yaml `spec.source.git.ref` default updates to chart or pelorus version?

Not always, we have 3 make targets:
```Makefile
release:
        ./scripts/create_release_pr

minor-release:
        ./scripts/create_release_pr -i

major-release:
        ./scripts/create_release_pr -m
```

Let's go over each of them:
```shell
$ make release

# This modifies the files so a new github tag (release) can be created. 
# It goes to the GitHub API and checks what is the last released version x.y.z and bumps the z
# To do so, only main Pelorus Chart requires to be modifed, and the file that references GIT new tag:
# charts/pelorus/subcharts/exporters/templates/_buildconfig.yaml
```

```shell
$ make release

# This modifies the files so a new github tag (release) can be created. 
# It goes to the GitHub API, checks what is the last released version x.y.z and bumps the z
# To do so, only main Pelorus Chart requires to be modifed, and the file that references GIT new tag, so it is
# NOT the Chart version but the github tag:
# charts/pelorus/subcharts/exporters/templates/_buildconfig.yaml
# We can not bump each individual Chart version nor we want as the source code of the individual 
# chart versions may not be the same.
# We want here to always bump z+1 and to sustain this we can not calculate z versions of individual Chart versions
# because they may have been bumped manually.
```

```shell
$ make minor-release

# This goes to the GitHub API, checks what is the last released version x.y.z and bumps the y
# This bumps all Chart versions, so we start from 0 in the z number
# In this case our GitHub release version will match Chart versions and all the numbers should be the same
```

```shell
$ make major-release

# This goes to the GitHub API, checks what is the last released version x.y.z and bumps the x
# It's same as minor-release but X is bumped and y,z are 0'ed
```

**Review by mateusoliveira43 — APPROVED** [review]

**Pull request merged**